cmd/pmemctl/pmemctl.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FILE=""
+FS=""
+MNT_DIR=""
+SIZE=""
+BLOCK_SIZE=4096
```

Comment:
You could do:

```sh
stat -c "%s" /
```

... and maybe default to 4096 if that fails?

Reply:
the block size for DAX *must be*  4096, otherwise it won't work